e2etest/pkg/service/create.go

```diff
@@ -36,6 +38,11 @@ func CreateWithBackend(cs clientset.Interface, namespace string, jigName string,
 	return svc, jig
 }
 
+func DeleteService(cs clientset.Interface, svc *corev1.Service) {
```

Comment:
Small nit: can you rename this `Delete` instead? We usually avoid repeating the package name in a func (calling service.DeleteService).